<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>secWindow</class>
 <widget class="QDialog" name="secWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>612</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KennelSoft</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Dashboard</string>
      </attribute>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>460</y>
         <width>441</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnCheckIn">
          <property name="text">
           <string>Check In</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCheckOut">
          <property name="text">
           <string>Check Out</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_hotelname">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>241</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Hotell navn</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Antall plasser </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_totelCages">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>50</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_freeCages">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>150</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>22</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Antall plasser hund</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_dog">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>25</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_freeDogCages">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>220</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>13</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Antall plasser katt</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_Cat">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>290</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>25</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_freeCatCages">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>290</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>15</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>40</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Hi!</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_username">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>40</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_tlf">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>380</y>
         <width>211</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Tlf</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_adr">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>410</y>
         <width>221</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Adresse</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Animal Overview</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLineEdit" name="inputSearch"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QTableView" name="tableView_Animals">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Search:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="tester">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
